pages/install/conda-install.rst

```diff
@@ -206,3 +206,80 @@ by running the commands:
 
    conda activate ct-dev
    conda update --channel cantera cantera cantera-matlab
+
+Development (C++ & Fortran 90) Interface
+========================================
+
+**Warning:** The ``conda`` package for Cantera's development interface is experimental.
+For the most recent version of the package, use the ``cantera/label/dev`` channel
+instead of the stable ``cantera`` channel.
+
+Cantera's development interface can be installed from the ``cantera`` channel. In this
+example, the command will create a new conda environment named ``ct-env``. From the
+command line (or the Anaconda Prompt on Windows), run:
+
+.. code:: shell
+
+   conda create --name ct-dev --channel cantera libcantera-devel
```

Comment:
```suggestion
   conda create --name ct-dev --channel cantera/label/dev libcantera-devel
```

At least until we feel comfortable removing the warning at the top of the section?

Reply:
Also added the description below.